graphql/resolvers.go

```diff
@@ -88,6 +91,24 @@ type queryResolver struct{ *Resolver }
 
 type patchResolver struct{ *Resolver }
 
+func (r *patchResolver) Duration(ctx context.Context, obj *restModel.APIPatch) (*PatchDuration, error) {
+	// excludes display tasks
+	tasks, err := task.Find(task.ByVersion(*obj.Id).WithFields(task.TimeTakenKey, task.StartTimeKey, task.FinishTimeKey))
```

Comment:
There's no need to manually dereference obj here

Reply:
according to the compiler i do need to dereference the obj here